README.md

```diff
@@ -447,6 +448,181 @@ you can create your own Configuration(s) and compose them with Config's ++ opera
+    Info : datacount=2 progbufsize=16
+    Info : Disabling abstract command reads from CSRs.
+    Info : Disabling abstract command writes to CSRs.
+    Info : [0] Found 1 triggers
+    Info : Examined RISC-V core; found 1 harts
+    Info :  hart 0: XLEN=64, 1 triggers
+    Info : Listening on port 3333 for gdb connections
+    Info : Listening on port 6666 for tcl connections
+    Info : Listening on port 4444 for telnet connections
+    
+### 5) Launch GDB
+
+Launch GDB in another terminal using
+
+    $ riscv64-unknown-elf-gdb helloworld
+    GNU gdb (GDB) 8.0.50.20170724-git
+    Copyright (C) 2017 Free Software Foundation, Inc.
+    License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
+    This is free software: you are free to change and redistribute it.
+    There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
+    and "show warranty" for details.
+    This GDB was configured as "--host=x86_64-pc-linux-gnu --target=riscv64-unknown-elf".
+    Type "show configuration" for configuration details.
+    For bug reporting instructions, please see:
+    <http://www.gnu.org/software/gdb/bugs/>.
+    Find the GDB manual and other documentation resources online at:
+    <http://www.gnu.org/software/gdb/documentation/>.
+    For help, type "help".
+    Type "apropos word" to search for commands related to "word"...
+    Reading symbols from ./proj1.out...done.
+    (gdb)
+
+Compared to Spike, the C Emulator is very slow, so several problems are encountered due to timeouts between issuing commands and response from the emulator. Thats why we should increase the `remotetimeout`.
+
+After that we load our program by performing a `load` command.
+
+ 	(gdb) set remotetimeout 2000
+ 	(gdb) target remote localhost:3333
+ 	Remote debugging using localhost:3333
+ 	0x0000000000010050 in ?? ()
+ 	(gdb) load
+ 	Loading section .text.init, size 0x2cc lma 0x80000000
+ 	Loading section .tohost, size 0x48 lma 0x80001000
+ 	Loading section .text, size 0x98c lma 0x80001048
+ 	Loading section .rodata, size 0x158 lma 0x800019d4
+ 	Loading section .rodata.str1.8, size 0x20 lma 0x80001b30
+ 	Loading section .data, size 0x22 lma 0x80001b50
+ 	Loading section .sdata, size 0x4 lma 0x80001b74
+ 	Start address 0x80000000, load size 3646
+ 	Transfer rate: 40 bytes/sec, 520 bytes/write.
+  	(gdb) 
+
+Now we can proceed as with Spike, debugging works in a similar way:
+
+	(gdb) target remote localhost:3333
+	Remote debugging using localhost:3333
+	0x000000008000104c in main (argc=0, argv=0x0) at src/main.c:16
+	16	    while (wait)
+	(gdb) print wait
+	$1 = 1
+	(gdb) print wait=0
+	$2 = 0
+	(gdb) print text
+	$3 = "Vafgehpgvba frgf jnag gb or serr!"
+	(gdb) c
+	Continuing.
+
+	^C
+	Program received signal SIGINT, Interrupt.
+	main (argc=0, argv=<optimized out>) at src/main.c:33
+	33	    while (!wait)
+	(gdb) print wait
+	$4 = 0
+	(gdb) print text
+	$5 = "Instruction sets want to be free!"
+	(gdb)
+
```

Comment:
I think you should add a note either here or at the beginning that will note what to do to make it waveform-enabled (building the `-debug` emulator target)

Reply:
Added in 1).
+ Pointed out the fact that VCD waveforms can be voluminous depending on program size.
+ I'll add a section for executable stripping may be? I guess it affects the execution time, do you agree @mwachs5 ?.

Reply:
Well, the program size and duration. I don't think it's really that relevant, people already know that VCD waveforms can be big and scale with runtime, so I woudl probably remove both those notes.

Reply:
You're right experienced people could already know this. But when it comes to beginners discovering all these, it is not the case. An example is a friend who tried to boot BBL (that has now several MiB of size) and you think it just doesn't work, but you find out that it just takes hours to run. In these cases gaining more space means gaining hours of waiting.
Anyway, I didn't test it personally. Is it the case normally?

Reply:
I mean, the larger program -> longer runtime is sort of orthogonal to longer runtime -> larger VCD file. So if you want to point out that it's going to be very slow to load a large program this way, I would separate it from the note that the generated waveforms with the `-debug` version of emulator may be large.

Reply:
I understand your point of view and I agree. 
As you said VCD size is trivial. Though, I pointed out the relation {larger program -> longer runtime} and how to reduce it as well as how to test the program on the emulator event after beginning debugging.
Again, I know these are basic ideas, but documenting can save new people some time to understand what's going on.
Thanks a lot.

Reply:
I guess it's okay now. Any other recommendations @mwachs5 ?